src/libraries/Native/AnyOS/System.IO.Compression.Native/entrypoints.c

```diff
@@ -0,0 +1,63 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#include <stdint.h>
+#include <string.h>
+
+// Include System.IO.Compression.Native headers
+#include "../zlib/pal_zlib.h"
+#include "../brotli/include/brotli/decode.h"
+#include "../brotli/include/brotli/encode.h"
+#include "../brotli/include/brotli/port.h"
+#include "../brotli/include/brotli/types.h"
+
+#ifndef lengthof
+#define lengthof(rg) (sizeof(rg)/sizeof(rg[0]))
```

Comment:
A nit - can you please name it countof ? We have the same kind of macro defined as _countof in coreclr PAL, but names starting with underscores are not supposed to be used out of compiler and platform headers, thus countof sounds like a good match.

Reply:
We have `lengthof` defined in coreclr, in fact several of them. And there is a lot of use.
I did not want to include files from coreclr in something that lives in libs, so I basically copied the existing pattern. 

Perhaps we should change the pal one to be `lenghtof` ?

Reply:
There are 299 lengthof usages, 350 _countof usages and 210 COUNTOF usages in the whole coreclr. More _countof usages in coreclr out of PAL than in PAL. But ok, if the lengthof is already used with such frequency, let's keep it.

Reply:
There are at least 5 other common names for this macro. The libraries mini pal has `ARRAY_SIZE` defined for this purpose: https://github.com/dotnet/runtime/blob/master/src/libraries/Native/Unix/Common/pal_utilities.h#L46 in case you would like locally consistent at least.

Reply:
I will keep `lengthof` then. We should consolidate this. 

I think `lengthof` is a good name though. In my view everything has size, unordered/sparse collections have counts and contiguous arrays have lengths.